test/invoice_test.rb

```diff
@@ -1,5 +1,6 @@
 require './test/test_helper'
 require './lib/invoice'
+require './lib/sales_engine'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.